apps/prairielearn/src/lib/externalGraderLocal.ts

```diff
@@ -64,21 +66,37 @@ export class ExternalGraderLocal {
         throw new Error('No external grading image specified.');
       }
 
+      if (question.external_grading_entrypoint == null) {
+        // The grader-host does not currently support a null entrypoint, and
+        // schema does not allow the entrypoint to be skipped if external
+        // grading options are provided. To avoid local code having a different
+        // behavior than production, we ensure that the local code throws an
+        // error if the entrypoint is null, which would also happen in
+        // production. A future implementation could allow the entrypoint to be
+        // skipped to use the default entrypoint from the image, but this would
+        // require equivalent changes to the grader-host.
```

Comment:
Might as well link to the issue since we have it.

```suggestion
        // The grader-host does not currently support a null entrypoint, and
        // schema does not allow the entrypoint to be skipped if external
        // grading options are provided. To avoid local code having a different
        // behavior than production, we ensure that the local code throws an
        // error if the entrypoint is null, which would also happen in
        // production. A future implementation could allow the entrypoint to be
        // skipped to use the default entrypoint from the image, but this would
        // require equivalent changes to the grader-host; see
        // https://github.com/PrairieLearn/PrairieLearn/issues/10940.
```